packages/model/src/-private/promise-many-array.ts

```diff
@@ -71,9 +71,17 @@ export default class PromiseManyArray {
         }
         return false;
       };
+      meta.addMixin = (mixin: object) => {
+        deprecate(`Do not use A() on an EmberData PromiseManyArray`, false, {
```

Comment:
@runspired  I had to put deprecation call here for both `hasMixin` and `addMixin` to satisfy the tests... Not sure if this is the correct approach...